charts/coredns/coredns-pod-delete/rbac.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: coredns-sa
+  name: coredns-pod-delete-sa
```

Comment:
Pls change the name of service Account in the whole file [ it have multiple occurrence here]